Tests/SwiftSyntaxTestSupportTest/SourceEditsTestUtilsTest.swift

```diff
@@ -0,0 +1,54 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import SwiftSyntax
+import _SwiftSyntaxTestSupport
+
+public class SourceEditsUtilTest: XCTestCase {
+  public func testDiffOfTwoStringsSimple() throws {
+    let s1 = "struct A { func f() {"
+    let s2 = "struct AA { func f() {"
```

Comment:
Just to have more descriptive names, could you name these strings `base` and `new` like in `getConcurrentEdits`? Same for all the other test cases.